app/routes/photos/GalleryPictureRoute.js

```diff
@@ -97,8 +99,12 @@ export default compose(
     mapStateToProps,
     mapDispatchToProps
   ),
-  prepare(({ match: { params } }, dispatch) =>
-    dispatch(fetchGalleryPicture(params.galleryId, params.pictureId))
+  prepare(
+    ({ match: { params } }, dispatch) =>
+      Promise.all[
+        (dispatch(fetchGalleryPicture(params.galleryId, params.pictureId)),
+        dispatch(fetch(params.galleryId)).catch())
```

Comment:
Why the empty `catch`? What happens if we get an error?

Reply:
Was a leftover from a copy paste, will remove it